### PR TITLE
Fix feature tests for AdminLoginPage

### PR DESCRIPTION
**Pull request by PrincTwilig**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

Fix feature tests for AdminLoginPage

## Summary of change

There is 4 tests for "admin sign in"
- login and password is correct
- login and password is incorrect
- password is incorrect
- login is incorrect

image

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Beerka1 — APPROVED** [review]

File: spec/features/admin_login_spec.rb [hunk 1]
```diff
@@ -2,25 +2,44 @@
...
-  #     visit '/admins/sign_in'
-  #     fill_in 'Email', with: 'wrong@email.com'
-  #     fill_in 'Password', with: admin.password
-  #     click_button 'Log in'
-  #     expect(page).to have_content 'Log in'
-  #   end
-  # end
+RSpec.feature 'AdminLogins', type: :feature do
+  describe 'sign in admin page', js: true do
+    subject { create(:user, :admin) }
```
Comment (opazyniuk) [inline spec/features/admin_login_spec.rb:7]: let

File: spec/features/admin_login_spec.rb [hunk 2]
```diff
@@ -2,25 +2,44 @@
...
+    context 'when sign in with correct login and password' do
+      it 'redirect to admin page' do
+        visit '/users/sign_in'
+        fill_in 'Email', with: subject.email
+        fill_in 'Password', with: subject.password
+        click_button 'Log in'
+        expect(page).to have_content 'Signed in successfully.'
+      end
+    end
```
Comment (opazyniuk) [inline spec/features/admin_login_spec.rb:16]: spaces between contexts

**Review by opazyniuk — CHANGES_REQUESTED** [review]
add tests on ukrainian version

**Review by KostiaFed — APPROVED** [review]

**Review by opazyniuk — CHANGES_REQUESTED** [review]
add this
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do

**Review by opazyniuk — APPROVED** [review]

**Pull request merged**